README.md

```diff
@@ -190,30 +192,25 @@ Further delegation of a capability with `nb` fields set MUST respect the `nb` fi
 ``` json
 [
```

Comment:
```suggestion
```

Reply:
No array?

Reply:
(Just loose objects?)

Reply:
Yeah I overlooked the details, my interpretation of discussion was https://github.com/ucan-wg/spec/pull/133